sdk/storage/Azure.Storage.DataMovement/src/StreamToUriJobPart.cs

```diff
@@ -268,66 +277,49 @@ private async Task<bool> CreateDestinationResource(long blockSize, long length,
         /// </summary>
         private async Task InitialUploadCall(long blockSize, long expectedLength, bool singleCall)
         {
-            try
+            if (singleCall)
```

Comment:
All I did here was remove the try/catch since I moved the try/catch to the calling method (it only had one caller).